source/crud/tests/v2/default-write-concern.yml

```diff
@@ -0,0 +1,413 @@
+data:
+  - _id: 1
+    x: 11
+  - _id: 2
+    x: 22
+collection_name: default_write_concern_coll
+database_name: default_write_concern_db
+tests:
+  - description: Aggregate with $merge
+    operations:
+      - object: collection
+        name: aggregate
+        arguments:
+          pipeline: &ref_0
+            - $match:
+                _id:
+                  $gt: 1
+            - $merge:
+                into: other_test_collection
+    expectations:
+      - command_started_event:
+          command:
+            aggregate: default_write_concern_coll
+            pipeline: *ref_0
+            writeConcern: null
```

Comment:
Just to confirm, this is leveraging the recently added [Fields that must NOT be present in Actual Documents](https://github.com/mongodb/specifications/tree/master/source/crud/tests#fields-that-must-not-be-present-in-actual-documents) section of the test README, correct?

Reply:
Correct. Noting in a comment.